docs/client/introduction.mdx

```diff
@@ -3,21 +3,60 @@ title: Client SDKs
+### 1. Initialization
 
-It may be helpful to review our documentation on a `StatsigUser` before integrating an SDK:
+- The client SDK's `initialize` call takes the **client SDK key** and a [**StatsigUser**](/client/concepts/user) object and makes a network request to Statsig servers; this network call fetches precomputed configuration parameters for the specified user from Statsig servers and stores these parameters in local storage on the client device
 
-- [StatsigUser](/client/concepts/user)
+-	Statsig's server latency to service `initialize` calls is generally 10ms (p50); the latency for given client may vary depending on how far the device is from Statsig's servers; the client SDK has a built-in timeout of 3 seconds that you can configure using **StatsigOptions** when you initialize the SDK
 
-Or, you can jump right in to the technical docs, by language, on the sidebar or below:
+-	The [**StatsigUser**](/client/concepts/user) object that you provide in the `initialize` call should include the user identifier, _userID_, that you use to identify the end-user of your application; the client SDK also generates a device identifier called _stableID_ to enable experiments where users aren’t signed in and a _userID_ is not available; you can choose to override this _stableID_ through **StatsigOptions** using the _overrideStableID_ parameter when you initialize the SDK
+
+:::info Using [**StatsigUser**](/client/concepts/user)
+
+Learn how to use `StatsigUser` before using a client SDK. 
+
+:::
+
+The configuration parameters received from Statsig servers in response to the `initialize` call persist until you either make another `initialize` call (recommended when the end-user starts a new session) or an `updateUser` call (recommended when an end-user attribute changes). These calls trigger a revaluation of all experiment configuration for the end-user.   
+
+### 2. Serving Variants
+
+ - After the `initialize` call completes, any subsequent `getExperiment` call synchronously returns the experiment configuration for the variant that your end-user is assigned to (you define these parameters when you set up an experiment on the Statsig console);  use the returned `config` object to `get` the value of the parameters required to implement the experiment variant
+ 
+ - If your experiment is part of a Layer, the `getLayer` call returns the experiment configuration for the variant that the your end-user is assigned to;  again, use the returned `config` object to `get` the value of the parameters required to implement the experiment variant
+  
+ - The `checkGate` calls returns _true_ if the end-user passes the feature gate rules that you've configured on the Statsig console, and _false_ otherwise 
+ 
+ - The `getConfig` call returns the dynamic configuration based on the rules that you've configured to target your end-users; use the returned `config` object to `get` the value of the parameters required to serve a dynamically configured app experience
+
```

Comment:
should we simplify all of this?  Just talk about gate or experiment or something?  Its a lot to process, and if I am knew, I dont know what to set up.  An experiment?  Does it need a layer? etc